src/screens/SignIn/SignInScreen.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import {
+  TouchableOpacity,
+  View,
+  Text,
+  TextInput,
+  Button,
+  ScrollView,
+  Linking,
+} from 'react-native'
+import { signIn } from 'src/store/actions'
+import { StackNavigationProp } from '@react-navigation/stack'
+import { styles } from './styles'
+
+type RootStackParamList = {
+  Home: undefined
+  SignIn: undefined
```

Comment:
Type 'undefined' looks a little bit strange here

Reply:
In react native navigation offered using "undefined" if we doesn't have params - "Specifying undefined means that the route doesn't have params. A union type with undefined (e.g. SomeType | undefined) means that params are optional." https://reactnavigation.org/docs/5.x/typescript

Reply:
Oh, great! Thanks for info 👍🏻